frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/align_toolbar_item/custom_text_align_command.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:appflowy/plugins/document/presentation/editor_plugins/plugins.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+
+final List<CommandShortcutEvent> customTextAlignCommands = [
+  customTextLeftAlignCommand,
+  customTextCenterAlignCommand,
+  customTextRightAlignCommand,
+];
+
+/// Windows / Linux : ctrl + shift + l
+/// macOs           : ctrl + shift + l
+/// Allows the user to align text to the left
+///
+/// - support
+///   - desktop
+///   - web
+///
+final CommandShortcutEvent customTextLeftAlignCommand = CommandShortcutEvent(
+  key: 'Align text to the left',
+  command: 'ctrl+shift+l',
+  handler: (editorState) => _textAlignHandler(editorState, leftAlignmentKey),
+);
+
+/// Windows / Linux : ctrl + shift + e
+/// macOs           : ctrl + shift + e
+/// Allows the user to align text to the center
+///
+/// - support
+///   - desktop
+///   - web
+///
+final CommandShortcutEvent customTextCenterAlignCommand = CommandShortcutEvent(
+  key: 'Align text to the center',
+  command: 'ctrl+shift+e',
+  handler: (editorState) => _textAlignHandler(editorState, centerAlignmentKey),
+);
+
+/// Windows / Linux : ctrl + shift + r
+/// macOs           : ctrl + shift + r
+/// Allows the user to align text to the right
+///
+/// - support
+///   - desktop
+///   - web
+///
+final CommandShortcutEvent customTextRightAlignCommand = CommandShortcutEvent(
+  key: 'Align text to the right',
+  command: 'ctrl+shift+r',
+  handler: (editorState) => _textAlignHandler(editorState, rightAlignmentKey),
+);
+
+KeyEventResult _textAlignHandler(EditorState editorState, String align) {
+  final Selection? selection = editorState.selection;
+
+  if (selection == null) {
+    return KeyEventResult.ignored;
+  }
+
+  // because the event handler is not async, so we need to use wrap the async function here
+  () async {
```

Comment:
There's no need to use 'async' here because this function is not asynchronous; thus, 'await' won't work.

Reply:
> There's no need to use 'async' here because this function is not asynchronous; thus, 'await' won't work.

The function `editorState.updateNode()` returns a `Future<void>`, so I encapsulated its invocation within an asynchronous immediately-invoked function expression `(() async {}())`. I used this construct to handle the asynchronous nature of the operation, ensuring proper execution. 

This same type of invocation is used in custom copy and paste command handlers

Reply:
I see. In this case, these two styles are acceptable.

I will merge it after you correct the macOS spelling.

Reply:
I've Implemented the changes as per your suggestions. Let me know if there's anything else you'd like to address or modify. Thanks!

Reply:
This sync block doesn't work how you expect it.

Don't await something if you're not using its value.

Take this example:
```

void main() {
  () async {
    await soup();
  }();
  
  print("reached end");
}

Future<void> soup() async {
  await Future.delayed(const Duration(seconds: 5), () {
    print("reached soup");
  });
}

```
What would you expect to happen? In reality reached end is printed first.

And then five seconds later approximately, reached soup is printed.

In the case above, the method returns handled key event before even finishing the update node. Rendering the asynchronous block useless.

Reply:
Apologies for the confusion! I forgot that although we're awaiting a function inside an IIFE async wrapper, it only halts the function's execution within the block and not the code outside of the wrapper's async block. The code outside the async wrapper continues to run regardless. Thanks for the clarification!